lib/Dialog/Dialog.js

```diff
@@ -153,6 +165,12 @@ module.exports = kind(
 	*/
 	create: function () {
 		Popup.prototype.create.apply(this, arguments);
+		if(this.bodyComponents) {
+			this.$.message.destroy();
```

Comment:
Hi @aarontam , you mean like this code? https://github.com/enyojs/moonstone/blob/2.6.0-dev/lib/Button/Button.js#L166